src/__tests__/imagings/requests/NewImagingRequest.test.tsx

```diff
@@ -131,7 +131,7 @@ describe('New Imaging Request', () => {
       const wrapper = await setup()
       const saveButton = wrapper.find(Button).at(0)
       expect(saveButton).toBeDefined()
-      expect(saveButton.text().trim()).toEqual('actions.save')
+      expect(saveButton.text().trim()).toEqual('Request Imaging')
```

Comment:
Rather than hard coding the text, could you make this compatible with language translation from t() function?

Reply:
Hey @aguptamusic ,

Just tried to resolve this issue in my latest commit. Thanks for catching that error!